journalpump/journalpump.py

```diff
@@ -495,6 +496,14 @@ def value_func(tags, f=f, args=args):  # pylint: disable=undefined-variable
 
             yield output
 
+    def _build_secret_filters(self, config):
+        secret_filters = config.get("secret_filters")
+        if secret_filters is not None:
+            for idx, _ in enumerate(secret_filters):
```

Comment:
Prefer `for secret_filter in secret_filters`.

Reply:
Agree everywhere else, but in this case I am modifying the original object. 
So, I could use this pattern + build a new object, or stick with the existing pattern. Any preference?

Reply:
In your example, i'd be modifying a copy of the list item, not the item in the original list. 
Here's a contrived example to show what i mean:
```
>>> list_of_items = [1, 2, 3, 4, 5]
>>> 
>>> # operating on a copy of the list item
>>> for item in list_of_items:
...     item = item + 1
... 
>>> print(list_of_items)
[1, 2, 3, 4, 5]
>>> 
>>> # operating on the original list object
>>> for idx, _ in enumerate(list_of_items):
...     list_of_items[idx] = list_of_items[idx] + 1
... 
>>> print(list_of_items)
[2, 3, 4, 5, 6]
```